### DISCUSSION TITLE
IOS build

### DISCUSSION THREAD

**Discussion by Vm900** [orbitalquark/textadept#76] [Q&A]
Hi,
Just wondering if its possible to compile it on iphone ?Thank you

**Comment by orbitalquark** [comment]
Sorry, it is not.

**Reply by Vm900** [reply]
Here’s how far i got 
![TextadeptBuildError](https://user-images.githubusercontent.com/9399631/111691748-87002300-8826-11eb-8cf4-c1cb7c52bd49.png)

**Reply by rgieseke** [reply]
Have you run `make deps`?

**Reply by Vm900** [reply]
yes

**Reply by Vm900** [reply]
Anyways found the problem.It was iphone blocking it because i was running out of space.Compiles perfectly with no warnings whatsoever.Thanks

**Comment by orbitalquark** [comment]
Thanks for following up. I didn't realize you were trying to compile the terminal version. I assumed you wanted the GUI version on iOS.
